<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>587</width>
    <height>337</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QComboBox" name="comboBox"/>
         </item>
         <item>
          <widget class="QComboBox" name="serial_port"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QPushButton" name="door1">
             <property name="text">
              <string>door1</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="door3">
             <property name="text">
              <string>door3</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="door5">
             <property name="text">
              <string>door5</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="door7">
             <property name="text">
              <string>door7</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QPushButton" name="door2">
             <property name="text">
              <string>door2</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="door4">
             <property name="text">
              <string>door4</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="door6">
             <property name="text">
              <string>door6</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="door8">
             <property name="text">
              <string>door8</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>221</width>
       <height>53</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="2">
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QPushButton" name="control_button">
       <property name="text">
        <string>开始</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="recoveryData">
       <property name="text">
        <string>复位</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="car1_front_fault">
       <property name="text">
        <string>tc1前方障碍物</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="car2_door2_fault">
       <property name="text">
        <string>2车前方障碍物</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
